apps/mark-scan/backend/src/app.test.ts

```diff
@@ -554,7 +554,7 @@ test('printing ballots', async () => {
   await waitForStatus('presenting_ballot');
   const interpretationChinese = await apiClient.getInterpretation();
   assert(interpretationChinese);
-  expect(interpretationChinese.type).toEqual('InterpretedBmdPage');
+  assert(interpretationChinese.type === 'InterpretedBmdPage');
```

Comment:
Moving the interpretation type assertion out of the API method meant that the `.votes` field access below started failing the type check (since jest `toEqual` doesn't provide compile-time assertion)